src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -131,15 +135,19 @@ export class GraphFilter extends React.PureComponent<GraphFilterProps> {
     return (
       <>
         <Toolbar>
-          {this.props.node && (
-            <FormGroup className={zeroPaddingLeft}>
-              <Button className={namespaceStyle} onClick={this.handleNamespaceReturn}>
-                Back to Full Graph...
-              </Button>
-            </FormGroup>
-          )}
-          <FormGroup className={zeroPaddingLeft}>
-            <GraphSettingsContainer edgeLabelMode={this.props.edgeLabelMode} graphType={this.props.graphType} />
+          <FormGroup className={breadcrumbStyle} style={{ ...thinGroupStyle }}>
```

Comment:
Using both the **className** and the **style** is probably not the best way to do this.

Reply:
So, I'm not too savvy with this stuff.  But when I tried to combine it into one className style, or all style properties, it just didn't seem to do what I want.  So in general I'm not using styles, but I probably should.  Instead I'm using this "style" objects and a spread operator.  I'd be happy to improve this but I may need help, or a PR to this PR.